### ISSUE TITLE
Inaccurate data lol

### ISSUE THREAD

**Issue by WTFBlaze** [OptoCloud/MonkeyList#1]
You grabbed the info of people who reacted to a ping. Good job lol want a cookie? Anyways I'm just here to point out that your so called "Monkey List" is incredibly inaccurate. That list does nothing except show who voted yes. Just cause someone voted yes doesn't mean they are an avatar leaker or an avatar ripper. My name got thrown in your list and (while I don't care if I'm gonna be completely honest here) I don't rip avatars. I can even show you my uploads if requested to prove my point about how this list in inaccurate.

If you wanted a genuine more accurate list you could attempt to do what HI did and make a ripper detector / flagger.

**Comment by LargestBoi** [comment]
He ain't smart enough to make a yoink detector for open source yoinkers, frfr. In other news it's literally a targeted harassment campaign against a set of discord users, no matter the statement of it being for "educational purposes only" man's risking his GitHub account to be a monkey himself.

**Comment by Kylie-Wes** [comment]
what's your discord? ^^

**Comment by OptoCloud** [comment]
@WTFBlaze I provided a clear explenation of the source of the data in the screenshot, you voted yes for a marketplace where any vrc content (not just ripped models) can be uploaded by users and shared (on a site that is already dedicated for ripping).

 which is suspiciously close to what another certain site is doing, and would enable sharing of unity packages. It's up to whoever looks at this repo to determine how accurate this data is lol.

So cope and get lost.

@LargestBoi I checked GitHub's TOS before posting this, love you too ❤️
